third_party/musl/src/include/errno.h

```diff
@@ -3,12 +3,16 @@
 
 #include "../../include/errno.h"
 
+// Prior to SB 16, ___errno_location was a weak alias for __errno_location in musl.
+// Starting with SB 16, we no longer implement __errno_location in musl. Instead, we retrieve the __errno_location symbols from the system.
+#if SB_API_VERSION < 16
```

Comment:
This is because "triple underscores" ___errno_location was a weak alias for "double underscores" __errno_location in musl.
and when we no longer defining "double underscores"__errno_location in musl, linker will complaining about undefined  "triple underscores"___errno_location, e.g.
ld.lld: error: undefined hidden symbol: ___errno_location
>>> referenced by mbrtowc.c:49 (/usr/local/google/home/haozheng/cobalt/out/evergreen-x64_devel/../../third_party/musl/src/multibyte/mbrtowc.c:49)
>>>               c_internal/mbrtowc.o:(mbrtowc) in archive obj/third_party/musl/libc_internal.a